### PR TITLE
Refactor 1.4.0

### PR DESCRIPTION
**Pull request by maximerety**

### CODE REVIEW

**Comment by dmerejkowsky** [comment]
how dare you refactoring without adding tests ?

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+indent_size = 2
+
+# it's recommend to keep these unchanged
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (dmerejkowsky) [inline .editorconfig]: trim_trailing_whitespace for markdown *is* a good idea

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# EditorConfig helps developers define and maintain consistent
```
Comment (dmerejkowsky) [inline .editorconfig:1]: remove useless comments please

**Review by dmerejkowsky — DISMISSED** [review]

**Comment by maximerety** [comment]
@dmerejkowsky 
> how dare you refactoring without adding tests ?


Use the shame Luke!
Fair enough, I was desperately trying to avoid yet another mocha, chai, ... setup but tests are coming, I promise.

File: .editorconfig (hunk 2 above)
Comment (maximerety) [reply .editorconfig:1]: @Marwyn42 and I disagree about the usefulness of this comment.

**Pull request merged**